app/authzed/concepts/workload-isolation/page.mdx

```diff
@@ -1,4 +1,4 @@
-import YouTube from "react-youtube";
+import YouTube from "@/components/youtube-wrapper";
```

Comment:
We needed to wrap this component because it's an old-style class component and next doesn't like them in server components, which is what these `mdx` files turn into.